### PR TITLE
Mark anonymizer as deprecated in favor of pseudomyze processor

### PR DESCRIPTION
**Pull request by MayeulRousselet**

### CODE REVIEW

File: anonymizer/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PLUGIN_VERSION=0.0.4
+PLUGIN_VERSION=0.0.5
```
Comment (Marsobad) [inline anonymizer/Makefile:1]: question (non blocking) : should we add a change log file? Could be useful for the plugin newsletter but yes not a must have

File: anonymizer/plugin.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "id": "anonymizer",
-    "version": "0.0.4",
+    "version": "0.0.5",
     "meta": {
-        "label": "Data anonymizer",
-        "description": "Preparation processor for anonymizing data",
+        "label": "Data anonymizer (deprecated)",
```
Comment (Marsobad) [inline anonymizer/plugin.json:5]: question (non blocking) : should we also specify that the plugin is deprecated in the help section of the processor (method getHelp) ?

**Review by Marsobad — APPROVED** [review]
LGTM and tested on a V12 kit ! Just a few non blocking questions

File: anonymizer/Makefile (hunk 1 above)
Comment (MayeulRousselet) [reply anonymizer/Makefile:1]: As the plugin is deprecated, I don't think we should invest more time on it

File: anonymizer/plugin.json (hunk 2 above)
Comment (MayeulRousselet) [reply anonymizer/plugin.json:5]: Good point. @chaokang worked on a feature to provide deprecation/replacement for processor. I don't know if we want to invest these effort nor if the system is compatible with plugins  (I could give a try)

Comment (MayeulRousselet) [reply anonymizer/plugin.json:5]: done in 3ee196ff12956dd9e8286c689b16215680ec6974

**Review by Marsobad — APPROVED** [review]
Great! LGTM

**Pull request merged**